### PR TITLE
only set setting on citusdb_master

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/citusdb/tasks/main.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@
      postgresql_shared_preload_libraries:
        - 'citus'
        - 'pg_stat_statements'
-     postgresql_custom_settings: "{{ [\"citus.use_secondary_nodes = 'always'\"] if is_pg_standby else []}}"
+     postgresql_custom_settings: "{{ [\"citus.use_secondary_nodes = 'always'\"] if is_pg_standby and 'citusdb_master' in groups else []}}"
```
Comment (rohit-dimagi) [inline src/commcare_cloud/ansible/roles/citusdb/tasks/main.yml]: i don't why but this evaluation condition is not working as it should be.

Comment (snopoke) [reply src/commcare_cloud/ansible/roles/citusdb/tasks/main.yml]: is there an error?

Comment (rohit-dimagi) [reply src/commcare_cloud/ansible/roles/citusdb/tasks/main.yml]: No error . it evaluates to true even for the citusworkerstandby node.

Comment (snopoke) [reply src/commcare_cloud/ansible/roles/citusdb/tasks/main.yml]:
```suggestion
     postgresql_custom_settings: "{{ [\"citus.use_secondary_nodes = 'always'\"] if is_pg_standby and inventory_hostname in groups['citusdb_master']|default([]) else []}}"
```

**Pull request merged**